appcues/src/main/java/com/appcues/data/model/ExperienceStepFormState.kt

```diff
@@ -18,6 +18,9 @@ internal class ExperienceStepFormState {
     val formItems: Collection<ExperienceStepFormItemState>
         get() = _formItems.values.sortedBy { it.index }
 
+    val lastTextInputItem: ExperienceStepFormItemState?
```

Comment:
the lastTextInputItem is useful to know whether we should use NEXT or DONE as imeAction [here](https://github.com/appcues/appcues-android-sdk/pull/260/files#diff-09087bb0e71de3f379426edc210622bd24fbcdeff8d12de15e0f27d3eeda986bR162)